concordium-consensus/src/Concordium/Scheduler/WasmIntegration/V1.hs

```diff
@@ -152,6 +155,10 @@ invokeResponseToWord64 (Error (EnvFailure e)) =
     MissingAccount _ -> 0xffff_ff02_0000_0000
     MissingContract _ -> 0xffff_ff03_0000_0000
     InvalidEntrypoint _ _ -> 0xffff_ff04_0000_0000
+    -- TODO: Consolidate on the error codes.
+    UpgradeInvalidModuleRef _ -> 1
+    UpgradeInvalidContractName _ _ -> 2
+    UpgradeInvalidVersion _ _ -> 3
 invokeResponseToWord64 MessageSendFailed = 0xffff_ff05_0000_0000
```

Comment:
As described above,  the environmet failure only uses the 4th byte, the first 3 are meant to be all set, and the last 4 all unset.